### ISSUE TITLE
keras ReduceLROnPlateau callback expects lr property

### ISSUE THREAD

**Issue by LaurensHagendoorn** [CyberZHG/keras-radam#2]
**Describe the Bug**
The keras callback ReduceLROnPlateau, expects a property called "lr" for the learning rate of the optimiser. RAdam does not appear to have this? (also, the parameter is called learning_rate? where normally it would be just lr)

**Comment by LaurensHagendoorn** [comment]
(btw, you implemented RAdam super fast! Nice! 👍 )

**Comment by CyberZHG** [comment]
The base optimizer is copied from the `tf-2` branch of keras. I've changed the name to `lr`.
